app/controllers/admin/conference_controller.rb

```diff
@@ -27,7 +27,6 @@ def index
       @recent_users = User.limit(5).order(created_at: :desc)
       @recent_events = Event.limit(5).order(created_at: :desc)
       @recent_registrations = Registration.limit(5).order(created_at: :desc)
-
```

Comment:
Shall we remove this file from the commit since there is no actual change to it?